recipes/tensorly-torch/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "tensorly-torch" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tensorly-torch-{{ version }}.tar.gz
+  sha256: 2b45f8bd1ea10a71f9332020ab36f5030ea9643812e16ff7572ab80c1d9160fe
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - nose
+    - numpy    
+    - scipy
+    - pytorch >=1.5.1
```

I think you need `tensorly` here too:

```suggestion
    - pytorch >=1.5.1
    - tensorly
```